DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/CourseDelegatesController.cs

```diff
@@ -41,8 +41,9 @@ ISearchSortFilterPaginateService searchSortFilterPaginateService
             this.searchSortFilterPaginateService = searchSortFilterPaginateService;
         }
 
-        [Route("{page:int=1}")]
+        [Route("{accessedVia}/{page:int=1}")]
         public IActionResult Index(
+            DelegateAccessRoute accessedVia,

```

Comment:
I don't really understand why you've done this access route stuff for this page. I don't think TopCourses should be a DelegateAccessRoute